test/parallel/test-tty-ref.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const common = require('../common');
+
+var TTY = process.binding('tty_wrap').TTY;
+var isTTY = process.binding('tty_wrap').isTTY;
+
+if (isTTY(1) == false) {
+  common.skip('fd 1 is not a tty.');
+  return;
+}
+
+var handle = new TTY(1);
+
+handle.unref();
+handle.ref(); // Should not throw.
```

Comment:
Please remove this test - it will always be skipped and is therefore not useful. :)